tutorials/mission_specific_analyses/nustar/analysis-nustar-lightcurve.md

```diff
@@ -76,11 +76,11 @@ except ModuleNotFoundError:
 
 We are interested in *NuSTAR* observation `60001110002`. To obtain the full path to the data directory, we can use [Xamin](https://heasarc.gsfc.nasa.gov/xamin/) and select `FTP Paths` in `Data Products Cart` to find the path:  `/FTP/nustar/data/obs/00/6//60001110002/`. 
```

Comment:
I had already [updated the notebook](https://github.com/HEASARC/heasarc-tutorials/blob/heasoftpy/heasoftpy/data-analysis-nustar.md) to use the simplified astroquery and more spectra extraction. It will be a good idea to consolidate these.